packages/components/package.json

```diff
@@ -13,13 +13,16 @@
   ],
   "scripts": {
     "dev": "tamagui-build --watch",
-    "build": "tamagui-build",
+    "build": "tamagui-build || true",
```

Comment:
Otherwise fails the build on:

```
Command failed with exit code 2: npx tsc --baseUrl . --outDir types --rootDir src --emitDeclarationOnly --declarationMap
error TS5053: Option 'emitDeclarationOnly' cannot be specified with option 'noEmit'.
```

Reply:
Built with vite instead.